Source/WebCore/css/typedom/DeclaredStylePropertyMap.cpp

```diff
@@ -65,25 +65,25 @@ auto DeclaredStylePropertyMap::entries(ScriptExecutionContext* context) const ->
     result.reserveInitialCapacity(declaredStyleSet.propertyCount());
     for (unsigned i = 0; i < declaredStyleSet.propertyCount(); ++i) {
         auto propertyReference = declaredStyleSet.propertyAt(i);
-        result.uncheckedAppend(makeKeyValuePair(propertyReference.cssName(), reifyValueToVector(propertyReference.value(), document)));
+        result.uncheckedAppend(makeKeyValuePair(propertyReference.cssName(), reifyValueToVector(RefPtr<CSSValue> { propertyReference.value() }, document)));
```

Comment:
could we use copyRef()?

Reply:
A raw pointer is returned (`StyleProperties::PropertyReference::value()`).